api/dao/liststorage.py

```diff
@@ -226,6 +226,10 @@ def get_fileinfo(self, _id, analysis_id, filename = None):
         else:
             return files
 
+    def get_analysis_label(self, _id, analysis_id):
+        _id = bson.ObjectId(_id)
+        return self._get_el(_id, {'_id': analysis_id}).get('label', 'No Label')
+
```

Comment:
As a general rule, we like to have storage classes return the document as close to the original state as possible and let the handler classes, which are much more specialized, handle those null cases. If there was a different handler class that would act differently when there is no label, it would wouldn't be able to tell from what gets returned from AnalysesStorage (someone could name their analysis "No Label", for example).

Sidenote: we do sometimes fill in missing fields with acceptable null values (like None, [] or {}) from the Storage classes, but we assume in this case a missing field and a false-y field are the same db state.